profile/profile_test.go

```diff
@@ -108,6 +108,21 @@ func TestParseError(t *testing.T) {
 	}
 }
 
+func TestParseConcatentated(t *testing.T) {
+	prof := testProfile1.Copy()
+	// Write the profile twice to buffer to create concatented profile.
+	buf := bytes.NewBuffer(nil)
```

Comment:
Nit: just "var buf bytes.Buffer"

Reply:
Tests failed after I'd made this change. I saw the following error:
```
panic: runtime error: invalid memory address or nil pointer dereference
	panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0x50a819]

```

I think I might need to keep this as is.

Reply:
@nolanmar511 #312 seems to pass tests for me so not sure what was the issue you saw. Unless you did something like `var buf *bytes.Buffer` which would of course would crash because it does not really create a buffer var.

Reply:
Yeah -- that was probably my problem.